fuzzers/aflplusplus_mopt/fuzzer.py

```diff
@@ -40,7 +40,7 @@ def fuzz(input_corpus, output_corpus, target_binary):
 
     flags = ['-L0']  # afl++ MOpt activation at once.
     flags += ['-prare']  # rare branch scheduling.
-    flags += ['-s666']  # fixed random seed.
+    flags += ['-s123']  # fixed random seed.
```

Comment:
I'm not sure I understand why the seed is changing but I have no objections to doing so.

Reply:
just because then it is the same as in /aflplusplus/ - no other reason and not really helping or hurting :)